special-interest-groups/sig-weir/README.md

```diff
@@ -0,0 +1,28 @@
+# Weir Special Interest Group(Weir SIG)
```

Comment:
A weir is a low barrier which is built across a river in order to control or direct the flow of water.
Project Weir mainly focuses on controlling SQL flows, which is similar to a weir built across TiDB clusters.

Reply:
Could you add this description to the SIG readme?

Reply:
> Could you add this description to the SIG readme?

done

Reply:
I think we should find a more commonly recognized or connected name for a SIG - instead of using a metaphor.  Maybe here we can use Proxy SIG, for example.

Reply:
How about `sig-gateway`?